task3.2.py

```diff
@@ -0,0 +1,14 @@
+'''Напишите программу, которая найдёт произведение пар чисел списка.
+Парой считаем первый и последний элемент, второй и предпоследний и т.д.
+Пример:
+- [2, 3, 4, 5, 6] => [12, 15, 16];
+- [2, 3, 5, 6] => [12, 15]'''
+
+lst1 = [2, 3, 4, 5, 6]
+lst2 = [2, 3, 5, 6]
+new_lst = []
+product =0
+for i in range(0, (len(lst2) + 1) // 2):
```

Comment:
```suggestion
for i in range((len(lst2) + 1) // 2):
```
Если вы не используете шаг, то ноль, можно не передавать явно.